showcase/app/templates/components/advanced-table.hbs

```diff
@@ -930,6 +930,32 @@
 
   <Shw::Divider @level={{2}} />
 
+  <Shw::Text::H3>Wide container with a narrow AdvancedTable</Shw::Text::H3>
+
+  <div class="shw-component-advanced-table-narrow-wrapper">
+    <Hds::AdvancedTable
+      @isSelectable={{true}}
+      @onSelectionChange={{this.onMultiSelectUsersSelectionChange__demo3}}
+      @model={{this.model.userData}}
+      @columns={{array (hash key="id" label="ID" width="150px") (hash key="name" label="Name" width="150px")}}
+      @hasStickyHeader={{true}}
+      @isStriped={{true}}
```

Comment:
[suggestion] to simplify things, you could remove these two (they're not strictly needed, for this example)
```suggestion
```